groundstation/samwise.ksy

```diff
@@ -0,0 +1,34 @@
+meta:
+  id: beacon_task
+  endian: be
+  encoding: UTF-8
+
+seq:
+  - id: name
+    type: strz
+    encoding: UTF-8
+    terminator: 0

```

Comment:
From here kaitai [docs](https://doc.kaitai.io/user_guide.html#:~:text=As%20this%20is%20a%20very%20common%20thing%2C%20there%E2%80%99s%20a%20shortcut%20for%20type%3A%20str%20and%20terminator%3A%200.%20One%20can%20write%20this%20as%3A) this `terminator: 0` does not look needed here?